rustfmt.toml

```diff
@@ -1,8 +1,2 @@
-write_mode = "Overwrite"
-fn_arg_indent = "Tabbed"
-fn_args_layout = "Block"
-where_pred_indent = "Visual"
-where_indent = "Visual"
-fn_return_indent = "WithWhereClause"
+fn_args_layout = "Tall"
```

Comment:
This is [the default](https://github.com/rust-lang/rustfmt/blob/rust-1.45/Configurations.md#fn_args_layout), so we don't need to specify it manually.

Reply:
I saw that. I thought it would be good to be explicit due to the other removals

Reply:
I don't much care either way :shrug: